server/channels/db/migrations/postgres/000115_user_reporting_changes.up.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE users ADD COLUMN IF NOT EXISTS lastlogin bigint NOT NULL DEFAULT 0;
```

Comment:
This is still not removed.

Reply:
Ah I see, yeah I meant that using props should not be a problem. But yes, in general agree with your thinking that if it's a core functionality, then it should be a dedicated column. No issues then :+1: .